### ISSUE TITLE
Too many coins?

### ISSUE THREAD

**Issue by Qu3nt174** [thomaswp/JunimoKartWeb#4]
Hey !

Seems like there are way too many coins (specially on the 1st level) 

Could be nice to reduce the amount (There are approximately 70 to 85 coins i think)

**Comment by Nitnino24** [comment]
I agree. It's not the worst thing in the world, but it would be nice to have a closer approximation of the actual game.

**Comment by thomaswp** [comment]
Thanks for the feedback. The code is from the game's source code, but it may be out of date compared to the modern version. I can't easily update this app anymore, so unfortunately I can't address your issue.
